video/hnm_decoder.cpp

```diff
@@ -226,7 +226,7 @@ void HNMDecoder::readNextPacket() {
 	}
 
 	// We use -1 here to discrimate a possibly empty sound frame
-	uint32 audioNumSamples = -1;
+	uint32 audioNumSamples = uint32(-1);
```

Comment:
This looks like a bug (even the description above mentions a negative number for an unsigned variable). Perhaps it would be better to use int32 here

Reply:
No, the fix is good. The comment is less good.
Please see -1 as UINT32_MAX or -1u.
It's really used like a marker.

Reply:
Ok, thanks